### ISSUE TITLE
An undesirable navigation when Far Manager is being launched with a passed path

### ISSUE THREAD

**Issue by cpp11nullptr** [FarGroup/FarManager#319]
#### Defect description
When Far Manager is being launched with a passed path to active or passive panel and there is a sub-directory with the same name as a last directory in the path Far Manager performs a navigation into that sub-directory and set it a current directory.
#### How to reproduce
```
D:\>mkdir test1
D:\>cd test1
D:\test1>mkdir test1
D:\test1>far . .
D:\test1\test1>
```
#### Actual behaviour 
Far Manager performs a navigation into sub-directory for both panels.
#### Expected behaviour
Far Manager sets only a path which it was launched with as a current directory.
#### Additional information
The attached patch (sorry, GitHub doesn't support .patch file extension) should fix that issue and I'm happy to open PR with these changes but I need a confirmation that it's actually a bug (as a comment around that functional is a bit confusing).
#### Attachments
[remove-sub-directory-navigation-patch.txt](https://github.com/FarGroup/FarManager/files/5737511/remove-sub-directory-navigation-patch.txt)

**Comment by alabuzhev** [comment]
That's a bug indeed. However, please don't fix bugs by simply removing the code.

Should be fixed in 814d7a9.

**Comment by cpp11nullptr** [comment]
Sure, @alabuzhev, I'm just not very familiar with codebase. I can confirm that it solves the issue, thanks!
